valohai/internals/distributed_config/network.py

```diff
@@ -0,0 +1,40 @@
+from typing import Any, Dict, List
+
+
+class Network:
```

Comment:
I'm not sure this needs to be a separate class..?

Reply:
it was a separate well structured object in the JSON hierarchy with different accompanied methods (`primary_local_ip` etc.) so why shouldn't it be?

Reply:
I don't see why the end user of this library needs to do `x.network.primary_local_ip` when `x.primary_local_ip` is pretty clear too 

Sure, it's a sub-dict in the JSON, but these objects don't need to be an 1:1 mapping to the raw data.

Reply:
yeah, we are not even currently 1:1 as the attributes in JSON `config` object (`distributed.json > .config`) is moved to the root level

should be fine to move one level down as suggested, I doubt the structure will be growing too much? thought we might want to retain it as there it was in the first place in the JSON to group all the networking related things